src/platforms/android/configuration/integrations/fragment.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Fragment
+caseStyle: camelCase
+supportLevel: production
+sdk: sentry.java.android.fragment
+categories:
+  - mobile
+---
+
+<Note>
+
+This is a preview API, available in `5.1.0-beta.1` and above. Features available in a preview API are still in progress and may have bugs. We recognize the irony. The API may be renamed, changed, or removed in a future version.
+
+</Note>
+
+The `sentry-android-fragment` library provides [Fragment](https://developer.android.com/jetpack/androidx/releases/fragment) support for Sentry via the [FragmentLifecycleIntegration](https://github.com/getsentry/sentry-java/blob/a5f30b43b1dad2634ce020809f3b52e0d564a22a/sentry-android-fragment/src/main/java/io/sentry/android/fragment/FragmentLifecycleIntegration.kt). The source can be found [on GitHub](https://github.com/getsentry/sentry-java/tree/main/sentry-android-fragment/src/main/java/io/sentry/android/fragment).
+
+On this page, we get you up and running with Sentry's Fragment Integration, so that it will automatically add a breadcrumb for each Fragment's lifecycle.
+
+## Install
+
+To add the Fragment integration, [manually initialize](/platforms/android/configuration/manual-init/) the Android SDK, then add the `sentry-android-fragment` dependency. Using Gradle:
+
+```groovy
+implementation 'io.sentry:sentry-android:5.1.0-beta.1'
+implementation 'io.sentry:sentry-android-fragment:5.1.0-beta.1'
```

Comment:
cant use the release registry here since betas don't go to release registry

Reply:
One work around is to set it as default value (`{{  registry("sentry.java.android.fragment", "5.1.0-beta.1") }}`)

Reply:
yes for `sentry-android-fragment` but not for `sentry-android` otherwise they'd have different versions, that's why hardcoded